### PR TITLE
Add Huawei P8 Lite [alice]

### PR DESCRIPTION
**Pull request by Aurnytoraink**

### CODE REVIEW

File: manifests/huawei_alice.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<remote name="huawei-p8-lite"
```
Comment (jbruechert) [inline manifests/huawei_alice.xml:3]: Please fix the indentation here. (Have a look at the other manifests for an example)

**Comment by ghost** [comment]
@Aurnytoraink do you have a working port?
Is https://github.com/Halium/projectmanagement/issues/78 fixed?

**Comment by Aurnytoraink** [comment]
> @Aurnytoraink do you have a working port?
> Is [Halium/projectmanagement#78](https://github.com/Halium/projectmanagement/issues/78) fixed?

Nope, I left the project

**Comment by ghost** [comment]
thanks for the response.
But sad to hear :disappointed:

**Comment by unxcepted** [comment]
Hello. I have to port P8 Lite because i need hybris, so i can try to resume this port

**Comment by Aurnytoraink** [comment]
> Hello. I have to port P8 Lite because i need hybris, so i can try to resume this port

Hi, even if I leave the project, you can still work on this port since the issue is open !

**Comment by ghost** [comment]
> Hello. I have to port P8 Lite because i need hybris, so i can try to resume this port

Do you need hybris or just "not-android"? Check https://github.com/Halium/projectmanagement/issues/78#issuecomment-646967077

**Comment by unxcepted** [comment]
>> Hello. I have to port P8 Lite because i need hybris, so i can try to resume this port
> 
> Do you need hybris or just "not-android"? Check [Halium/projectmanagement#78 (comment)](https://github.com/Halium/projectmanagement/issues/78#issuecomment-646967077)

The funfact is that pmos port is mine
i need hybris image for it, so i can try to get some things to work @microsoft-sucks

**Comment by ghost** [comment]
> The funfact is that pmos port is mine
> i need hybris image for it, so i can try to get some things to work @microsoft-sucks

Now I recognize your name :sweat_smile: 

But hybris [isn't really wanted from pmOS](https://wiki.postmarketos.org/wiki/Hybris). Do you think mainlining could work somewhat better?

**Comment by unxcepted** [comment]
Ofc it would be better, but I don't know if I could do it. Now i just  want to make it work fully

**Comment by Vince1171** [comment]
Hi, thank for your interest in the Halium project
is this PR still relevant/up-to-date?
if yes, please said so
if nothing is done in a week, I'll close this PR

**Comment by Vince1171** [comment]
No response after a week
I close the PR

**Pull request closed without merging**